src/app/COMPONENTS/task-details/task-details.component.ts

```diff
@@ -35,11 +35,15 @@ import * as userSelectors from '../../store/user/user.selectors'
 export class TaskDetailsComponent implements OnInit {
   users$ = this.store.select(userSelectors.selectAllUsers)
   currentUserRoles$ = this.store.select(userSelectors.selectCurrentUserRoles)
+  selectedTask$ = computed(() => this.tasksSignal.selectedTaskSignal())
```

Comment:
is this an observable? Don't forget to add types to variables, otherwise this $ symbol is making me think it's an observable but in line 44 you're directly accessing the title field of this task, so it's probably not an observable